docs/recommendations/RNA/adjoined_transcript.md

```diff
@@ -0,0 +1,41 @@
+# Adjoined Transcripts
+
+<!-- ## Definition -->
+
+Adjoined Transcript: a transcript (RNA molecule) composed of adjoined RNA from two or more contributing transcripts.
+
+## Syntax
+
+```sh exec="true"
+bin/pull-syntax -c -f docs/syntax.yaml rna.adjoined_transcript
+```
+
+## Notes
+- Adjoined transcripts are a product of chimeric transcript gene fusions.
```

Comment:
Can we drop "chimeric transcript"?

Reply:
We can replace "chimeric transcript" with "some" to simplify this.

Reply:
addressed in 2e6dd27